src/nvim/buffer_defs.h

```diff
@@ -789,16 +789,16 @@ struct diffblock_S {
 typedef struct tabpage_S tabpage_T;
 struct tabpage_S {
   handle_T handle;
-  tabpage_T *tp_next;      ///< next tabpage or NULL
-  frame_T *tp_topframe;    ///< topframe for the windows
-  win_T *tp_curwin;        ///< current window in this Tab page
-  win_T *tp_prevwin;       ///< previous window in this Tab page
-  win_T *tp_firstwin;      ///< first window in this Tab page
-  win_T *tp_lastwin;       ///< last window in this Tab page
-  long tp_old_Rows_avail;  ///< ROWS_AVAIL when Tab page was left
-  long tp_old_Columns;     ///< Columns when Tab page was left, -1 when
-                           ///< calling win_new_screen_cols() postponed
-  long tp_ch_used;         ///< value of 'cmdheight' when frame size was set
+  tabpage_T *tp_next;         ///< next tabpage or NULL
+  frame_T *tp_topframe;       ///< topframe for the windows
+  win_T *tp_curwin;           ///< current window in this Tab page
+  win_T *tp_prevwin;          ///< previous window in this Tab page
+  win_T *tp_firstwin;         ///< first window in this Tab page
+  win_T *tp_lastwin;          ///< last window in this Tab page
+  int64_t tp_old_Rows_avail;  ///< ROWS_AVAIL when Tab page was left
+  long tp_old_Columns;        ///< Columns when Tab page was left, -1 when
```

Comment:
```suggestion
  int64_t tp_old_Columns;        ///< Columns when Tab page was left, -1 when
```

Might as well change this for consistency's sake